### DISCUSSION TITLE
Radeon Rx 560

### DISCUSSION THREAD

**Discussion by aladrocMatiner** [ROCm/ROCm#3145] [Q&A] [answered]
Hej, Im trying to install rocm with rx 560 and following some tips here and there seems i need to have the legacy drivers that are available on rocm 5.7.1 but I get this message on the installation:

`amdgpu-install --usecase=graphics,opencl --opencl=rocr,legacy --vulkan=amdvlk,pro
WARNING: legacy OpenCL is deprecated and will be removed soon.
INFO: i386 architecture has not been enabled with dpkg.
Installation of 32-bit run time has been excluded.
Hit:1 http://security.ubuntu.com/ubuntu jammy-security InRelease
Hit:2 http://se.archive.ubuntu.com/ubuntu jammy InRelease
Hit:3 https://repo.radeon.com/amdgpu/5.7.1/ubuntu jammy InRelease
Hit:4 http://se.archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:5 http://se.archive.ubuntu.com/ubuntu jammy-backports InRelease
Hit:6 https://repo.radeon.com/rocm/apt/5.7.1 jammy InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package clinfo-amdgpu-pro
E: Unable to locate package opencl-legacy-amdgpu-pro-icd
E: Unable to locate package vulkan-amdgpu-pro`

from rocm-smi i get nothing...
`rocm-smi

========================= ROCm System Management Interface =========================
WARNING: No AMD GPUs specified
=================================== Concise Info ===================================
GPU  Temp (DieEdge)  AvgPwr  SCLK  MCLK  Fan  Perf  PwrCap  VRAM%  GPU%
====================================================================================
=============================== End of ROCm SMI Log`

Wonder what im doing wrong?

**Comment by tcgu-amd** [comment, marked as answer]
Hi @aladrocMatiner, sorry for the lack of response. Could you try using amdgpu-install version 5.4.1?
 (https://repo.radeon.com/amdgpu-install/5.4.1/)
Hope this helps. Thanks!
